client/src/questions-n-answers/qNa.jsx

```diff
@@ -551,7 +580,8 @@ class QuestionsNAnswers extends React.Component {
                       addAnswerScroll={this.addAnswerScroll}
                       answerTableHide={this.answerTableHide}
                       answerHide={this.answerHide}
-                      // showButton={this.showButton}
+                      answerHide2={this.answerHide2}
```

Comment:
this is a text function i was using to try to solve a an answer displaying issue